.github/workflows/build/osx/action.yml

```
@@ -10,35 +10,24 @@ runs:
     run: |
       set -ex
       brew update-reset
-      brew install ninja
-
-  - name: Download HDF5 Artifacts
-    shell: bash
-    run: |
-      wget https://github.com/disorderedmaterials/HDF5/releases/download/${{ env.hdf5tag }}/${{ env.hdf5tag }}-osx.zip
-      unzip ${{ env.hdf5tag }}-osx.zip
+      brew install ninja hdf5 zlib
 
   - name: Build
     shell: bash
     run: |
       set -ex
-      HDF5_DIR="$(pwd)/${{ env.hdf5tag }}"
       mkdir build
       cd build
-      cmake ../ -G Ninja -DCMAKE_Fortran_COMPILER:string="gfortran-11" -DLOCAL_STATIC_HDF5:bool=True -DHDF5_DIR:path=${HDF5_DIR} -DCMAKE_Fortran_FLAGS:string="-cpp" -DGUDPY_COMPATIBILITY=1 ${{ inputs.extraflags }}
+      cmake ../ -G Ninja -DCMAKE_Fortran_COMPILER:string="gfortran-11" -DBUILD_SZIP:bool=true -DCMAKE_OSX_ARCHITECTURES:string="${{ env.TARGET }}" -DCMAKE_Fortran_FLAGS:string="-cpp" -DGUDPY_COMPATIBILITY=1 ${{ inputs.extraflags }}
       ninja
       ninja install
-
-  - name: Create Zip
-    shell: bash
-    run: |
-      set -ex
-      mkdir packages
-      mv bin binaries-${{ env.gudrunVersion }}-osx
-      zip -9rv packages/binaries-${{ env.gudrunVersion }}-osx.zip binaries-${{ env.gudrunVersion }}-osx/
+      cd ..
+      mkdir osx
+      mv bin osx/binaries-${{ env.gudrunVersion }}-osx-${{ env.TARGET }}
```

Putting these into a subdir `osx/` is unnecessary, you can just rename and use a wildcard in the `download-artifact` action.
```suggestion
      mv bin binaries-${{ env.gudrunVersion }}-osx-${{ env.TARGET }}
```